### DISCUSSION TITLE
How can i install Mta on my Mac os x computer?

### DISCUSSION THREAD

**Discussion by Panda4uOnly** [multitheftauto/mtasa-blue#2225] [Q&A] [answered]
So, i want to play "Mta" but i don't know how to download it on my Mac os x, pls help me)))

**Comment by Disinterpreter** [comment]
https://www.parallels.com/ for example, but if you have M1 processor that's impossible at the moment.

**Comment by Dutchman101** [comment, marked as answer]
Refer to [instructions: MTA:SA on macOS](https://gist.github.com/qaisjp/778ed99ac8a8dcb93f797a28ba5f8303) or even [https://wiki.multitheftauto.com/wiki/MacOS](https://wiki.multitheftauto.com/wiki/MacOS) if you're an advanced user

**Comment by MatinFree** [comment]
If you are a professional, refer to the instructions: MTA: SA on macOS or even https://wiki.multitheftauto.com/wiki/MacOS
There was another help in my service
